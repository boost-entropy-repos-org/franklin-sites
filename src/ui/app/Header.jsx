import React from 'react';
import {
  Link,
} from 'react-router-dom';

const Header = () => (
  <div className="header">
    <h3>Franklin</h3>
    <ul className="">
      <li>
        <Link to="/">Home</Link>{' '}
      </li>
      <li>
        <Link to="/ui-components">UI components</Link>{' '}
      </li>
    </ul>
  </div>
);

export default Header;

