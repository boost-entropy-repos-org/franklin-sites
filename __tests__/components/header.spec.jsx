import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../../src/components/header';

describe('Hader component', () => {
  test('should render', () => {
    const component = renderer.create(<Header />).toJSON();

    expect(component).toMatchSnapshot();
  });
});
