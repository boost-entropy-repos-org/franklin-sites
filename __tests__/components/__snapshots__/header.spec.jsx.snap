// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hader component should render 1`] = `
<div
  className="header"
/>
`;
